### PR TITLE
Build one library!

### PR DESCRIPTION
**Pull request by davidbeckingsale**

### CODE REVIEW

**Comment by mcfadden8** [comment]
This sounds like a great idea @davidbeckingsale!  The builds are currently not passing so I will wait until after they are fixed before reviewing.

**Comment by davidbeckingsale** [comment]
@mcfadden8 - builds are passing 😁

File: src/umpire/op/CMakeLists.txt [hunk 1]
```diff
@@ -28,8 +28,7 @@ set (umpire_op_sources
   MemoryOperation.cpp
   MemoryOperationRegistry.cpp)
 
-set (umpire_op_depends
-  umpire_util)
+set (umpire_op_depends)
```
Comment (mcfadden8) [inline src/umpire/op/CMakeLists.txt]: So is this just setting umpire_op_depends to nothing?  Should this and the reference line below be removed?

Comment (davidbeckingsale) [reply src/umpire/op/CMakeLists.txt]: No, because if CUDA or ROCm are used, they are added to the dependency list.

**Pull request merged**